Modules/core/BRWallet.c

```diff
@@ -620,6 +709,21 @@ BRTransaction *BRWalletCreateTxForOutputs(BRWallet *wallet, const BRTxOutput *ou
         transaction->asset->nameLen = asset->nameLen;
         transaction->asset->amount = asset->amount;
         transaction->asset->type = asset->type;
+        transaction->asset->reissuable = asset->reissuable;
+        transaction->asset->unit = asset->unit;
+        transaction->asset->hasIPFS = asset->hasIPFS;
+        if(transaction->asset->hasIPFS == 1)
```

Comment:
This seems backwards.
```suggestion
        if(asset->hasIPFS == 1)
```